src/components/utils/LinkHelper.tsx

```diff
@@ -1,27 +1,23 @@
 import * as React from "react";
-import Link from '@material-ui/core/Link';
-import Button from '@material-ui/core/Button';
-import CardActionArea from '@material-ui/core/CardActionArea';
-import { LinkProps, NavLinkProps } from 'react-router-dom';
+import Link from "@material-ui/core/Link";
+import Button from "@material-ui/core/Button";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import { LinkProps, NavLinkProps } from "react-router-dom";
+import { FunctionComponent } from "react";
 
-
-export const MaterialLinkRouter = (props: LinkProps) => {
-  return (
-    <Link {...props} />
-  );
+interface IMaterialLinkRouter extends LinkProps {
+  component: any;
```

Comment:
I'm missing something, why this prop?

Reply:
It's an old thing. I don't know why it's there too. I think the original idea was to have a customizable MaterialLinkRouter.